components/install/installSection.js

```diff
@@ -29,8 +32,81 @@ const steps = [
   }
 ]
 
+const InstallInstructions = ({ token }) => (
+  <Card
+    shadowSz='lg'
+    maxW='55rem'
+    margin='0 auto 3rem'
+    textAlign='left'
+  >
+    <Text marginBottom='3rem'>
+        Copy and paste the applicable command in your shell to continue:
+    </Text>
+    <InstallCommands token={token} />
+  </Card>
+)
+
+InstallInstructions.propTypes = {
+  token: PropTypes.string.isRequired
+}
+
+const PostInstallInstructions = ({ noAds }) => {
+  const topText = noAds ? "There's nothing else you need to do." : "Let's try it out!"
```

Comment:
i dont like "There is nothing else you need to do"

what about: "Thanks for choosing to support the Open Source community. This is a big day" or something more exciting / congratulatory